Sources/Prometheus/MetricTypes/Counter.swift

```diff
@@ -73,11 +73,10 @@ public class Counter<NumType: Numeric, Labels: MetricLabels>: Metric, Prometheus
                 var val = self.metrics[labels] ?? self.initialValue
                 val += amount
                 self.metrics[labels] = val
-                done(val)
             } else {
                 self.value += amount
-                done(self.value)
             }
+            done(self.value)
```

Comment:
Could you revert this to how it was? This way if you increment a specific set of labels, you get back a value of the base counter, not the labeled one

Reply:
Yikes, great catch! Thanks